firefox_puppeteer/api/appinfo.py

```diff
@@ -6,17 +6,38 @@
 
 
 class AppInfo(BaseLib):
+    """This class provides access to various attributes of AppInfo.
+
+    For more details on AppInfo, visit:
+    https://developer.mozilla.org/en-US/docs/Mozilla/QA/Mozmill_tests/Shared_Modules/UtilsAPI/appInfo
+    """
+
+    def __getattr__(self, attr):
+        with self.marionette.using_context('chrome'):
+            value = self.marionette.execute_script("""
+              return Services.appinfo[arguments[0]];
+            """, script_args=[attr])
+
+            if value is not None:
+                return value
+            else:
+                raise AttributeError(
+                    "{} has no attribute {}".format(self.__class__.__name__, attr))
```

Comment:
It looks like that you were able to learn all the necessary bits in git quiet fast. That's good to see. The update of this PR looks pretty good!

There is only one nit here, where you make use of double quotes instead of single quotes. But let me fix that when I merge your patch.